packages/localizations/src/index.ts

```diff
@@ -1,6 +1,6 @@
 export { enUS } from './en-US';
 export { frFR } from './fr-FR';
-export { deDe } from './de-DE';
+export { deDE } from './de-DE';
```

Comment:
🔧  This change is consider a breaking change since you change the exports of the package. To avoid breaking those using `import { deDe } from '@clerk/localizations` i would suggest to keep the current localization, add a deprecation warning and export the one with the correct key. Eg
```
/*
 * @deprecated use `deDE` instead
 */
export { deDe } from './de-DE';
export { deDE } from './de-DE';
```

Reply:
Makes sense! Done.

Reply:
Can't unsee now. 
Thanks for spotting that @harrisrobin :)